### PR TITLE
install eslint, prettier and gherkin-lint

### PR DESCRIPTION
**Pull request by islamov089**

### CODE REVIEW

File: config.js
Comment (jkalandarov) [inline config.js:1]: which config file is it?

File: package.json
Comment (jkalandarov) [inline package.json:1]: Have you added scripts to run eslint, gherkin-lint checks? and prettier script to fix issues?

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: config.js
Comment (islamov089) [reply config.js:1]: This file belongs to the Allure reporter. I have not finished yet.

File: package.json
Comment (islamov089) [reply package.json:1]: yes I just added and tested.

File: config.js
Comment (jkalandarov) [reply config.js:1]: please rename the file as `allure.config.ts` then (or .js).

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: config.js
Comment (islamov089) [reply config.js:1]: done

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**